packages/pretty-format/src/collections.js

```diff
@@ -162,7 +162,10 @@ export function printObjectProperties(
 ): string {
   let result = '';
   let keys = Object.keys(val).sort();
-  const symbols = getSymbols(val);
+  const symbols = getSymbols(val).filter(
+    //$FlowFixMe because property enumerable is missing in undefined
+    symbol => Object.getOwnPropertyDescriptor(val, symbol).enumerable,
```

Comment:
I think this should this be a part of `getSymbols`, since it's only used in one place